vividus-engine/src/main/java/org/vividus/model/jbehave/AbstractStepsContainer.java

```diff
@@ -55,8 +56,19 @@ public void setAfterUserScenarioSteps(List<Step> afterScenarioSteps)
         this.afterUserScenarioSteps = afterScenarioSteps;
     }
 
+    public List<Step> getAfterSystemScenarioSteps()
+    {
+        return afterSystemScenarioSteps;
+    }
+
+    public void setAfterSystemScenarioSteps(List<Step> afterSystemScenarioSteps)
+    {
+        this.afterSystemScenarioSteps = afterSystemScenarioSteps;
+    }
+
     public Stream<Step> createStreamOfAllSteps()
     {
-        return Stream.of(getBeforeUserScenarioSteps(), getSteps(), getAfterUserScenarioSteps()).flatMap(List::stream);
+        return Stream.of(getBeforeUserScenarioSteps(), getSteps(), getAfterUserScenarioSteps(),
+                getAfterSystemScenarioSteps()).flatMap(List::stream);
```

Comment:
why not to add `beforeSystemScenarioSteps`?

Reply:
there is nothing that can be failed